### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by turn1a** [TheR1D/shell_gpt#577] [General]
Hi,

Thanks for the awesome project, I've been enjoying using it tons! I see that there are 17 pull requests (as of writing this post), some dating back to December 2023, and most don't have any replies. What is the current development status of the project? Is it actively maintained? It currently has over 8k starts, which means it's quite popular, and it would be great to improve it even further by including some of the work that the community is putting into it.

Thanks!

**Comment by TheR1D** [comment]
Hi @turn1a,

Thank you for your interest, and I'm glad you find ShellGPT useful! The project receives regular updates, with the most requested features being added on a monthly basis. Regarding the pull requests, some of them do not align with the rules defined in [CONTRIBUTING.md](https://github.com/TheR1D/shell_gpt/blob/main/CONTRIBUTING.md), such as working on existing issues or gathering feedback for a particular feature the author wants to merge. Therefore, it takes longer to review and integrate these contributions. Also, since some features require a specific OS or a specific shell which I don't have, it becomes a bit time-consuming to properly test them, especially since most of the pull requests are reviewed by me.

I would like to encourage everyone to review or share their opinions, which will help to integrate pull requests. Your feedback and reviews are valuable, as they not only speed up the process but also ensure that we're including features and fixes that truly benefit the community.

**Comment by turn1a** [comment]
Hi @TheR1D,

Forgive me for my hasty judgment. Thanks for explaining! I now understand why some PRs are pending. I'll do my best to make mine easier for you. I'll also try to review a couple of others and provide feedback.
